pyracing/client.py

```diff
@@ -214,12 +213,15 @@ async def current_seasons(
             'carid': car_id,
             'seasonid': season_id,
             }
+        # Adds the key name to key_list if set to True
+        key_list = [key for key in field_dict if field_dict.get(key)]
 
-        key_list = self.key_list_from_dict(field_dict)
+        # iRacing accepts these as a single, comma seperated, parameter
+        key_list = ','.join(map(str, key_list))
```

Comment:
This should already be a list of strings, so i think `key_list = ','.join(key_list)` should work fine here